### PR TITLE
ARUHA-450: decreased DB connections to 2

### PR DESCRIPTION
**Pull request by adyach**

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -19,9 +19,14 @@ spring:
     password: nakadi
     driverClassName: org.postgresql.Driver
     initialSize: 2
-    maxActive: 20
+    maxActive: 2
```
Comment (v-stepanov) [inline src/main/resources/application.yml]: this property will be overridden with senza env variables we have in our private repo

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -19,9 +19,14 @@ spring:
...
+    minIdle: 1
+    maxIdle: 1
     testOnBorrow: true
     validationQuery: SELECT 1
+    testWhileIdle: true
+    timeBetweenEvictionRunsMillis: 5000
+    minEvictableIdleTimeMillis: 60000
```
Comment (v-stepanov) [inline src/main/resources/application.yml]: What these three properties are for?

File: src/main/resources/application.yml (hunk 1 above)
Comment (adyach) [inline src/main/resources/application.yml]: @v-stepanov changed

Comment (antban) [inline src/main/resources/application.yml]: Lets change initial size too.

Comment (antban) [inline src/main/resources/application.yml]: Ignore my previous comment

**Comment by antban** [comment]
:+1:

**Pull request merged**